04-materials/06-working-on-your-own.md

```diff
@@ -17,50 +17,241 @@ We will be using the [official template](https://github.com/jupyterlab/extension
-| `ISettingRegistry` | Extension settings | [Docs](https://jupyterlab.readthedocs.io/en/latest/api/modules/settingregistry.html) |
+Create something entirely new using the extension template, AI assistance, and the patterns you've learned.
+
+### ⚡ Quick start
+
+Follow these sections from chapter 2 to start a new extension from a template:
+
+1. [🛠️ Setup](02-anatomy-of-extensions.md#️-setup) - Set up your environment, Git, and GitHub repository
+2. [🏋️ Exercise A: Extension creation and development loop](02-anatomy-of-extensions.md#️-exercise-a-15-minutes-extension-creation-and-development-loop) - Create your extension from the template and test the development loop
+
+Once you have your extension set up and working, return here to choose an inspiration project below.
+
+### 💫 Inspiration
+
+The following three examples with prompts have been tested and confirmed working. Pick one that matches your interests - whether you want visual effects, creative customization, or full-stack development. Use them as-is to practice prompt engineering, or as inspiration for your own ideas.
+
+#### 🎉 Confetti celebration button
+
+**Complexity**: Beginner (Frontend only)
+**Time**: 15-20 minutes
+**What you'll learn**: Status bar integration, DOM manipulation, visual effects
+
+**The Prompt**:
+```
+Add a button in the status bar to celebrate something! It should show confetti
+on top of the UI for 3 seconds
+```
+
+**Expected Results**:
+- Status bar shows a celebration button (usually bottom-right)
+- Clicking triggers confetti animation overlay
+- Animation appears on top of all UI elements
+- Confetti clears after ~3 seconds
+- Multiple clicks work correctly
+
+**What to Watch For**:
+
+If the confetti doesn't appear:
+```
+The button is there but I don't see confetti. Check that the z-index
+is high enough and the confetti container is properly positioned.
+```
+
+To customize:
+```
+Make the confetti more colorful and add a sound effect when it triggers.
+```
+
+**Verified Setup**:
+- ✅ macOS 15.7, Claude Code (Claude Sonnet 4.5)
+- ✅ `conda` environment: Python 3.13, Node.js 22, JupyterLab
+- 📹 [Watch the demo](https://www.loom.com/share/2afabea0184045fa868271f9ab0ca083)
+
+:::{tip}
+This is a perfect first project - it's self-contained, purely visual, and you'll immediately see if it works. It teaches you about JupyterLab's status bar API without backend complexity.
+:::
+
+#### 🎨 Custom theme extension
+
+**Complexity**: Beginner (Frontend only)
+**Time**: 20-30 minutes
+**What you'll learn**: Theme customization, CSS styling, JupyterLab theming system
+
+**The Prompt**:
+```
+Create a theme based on [your favorite movie/show/game/aesthetic]
+```
+
+**Example**:
+```
+Create a theme based on Netflix show KPop Demon Hunters
+```
 
+**Expected results**:
+- A new theme appears in Settings → Theme menu
+- Theme includes custom colors matching your chosen aesthetic
+- JupyterLab interface reflects the theme's visual style
+- Theme can be toggled on/off
```

Comment:
```suggestion
- Theme can be toggled on/off in Settings → Theme menu
```

Or we can omit this since bullet 1 already mentions the theme being selectable in this menu.